rpc/lib/server/handlers.go

```diff
@@ -648,6 +652,10 @@ func (wsc *wsConnection) readRoutine() {
 				wsc.WriteRPCResponse(types.RPCInternalError(request.ID, errors.Wrap(err, "Error converting json params to arguments")))
 				continue
 			}
+			if len(args) == 0 && len(rpcFunc.args) > 0 {
+				wsc.WriteRPCResponse(types.RPCInternalError(request.ID, errors.New("Invalid number of params")))
```

Comment:
Maybe more appropriate to use `RPCInvalidParamsError` here instead of `RPCInternalError`?